src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,110 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int GROWING_NUMBER = 5;
+    private static final int FIRST_INDEX = 0;
+    private static final int DEFAULT_LENGTH = 10;
+    private int size = 0;
+    private T[] elementData;
+    private T element;
+
+    public ArrayList() {
+        elementData = (T[]) new Object[DEFAULT_LENGTH];
+    }
+
+    public T[] growIfArrayFull() {
```

Comment:
Private ones (methods, classes) should be after public ones.
```suggestion
    private T[] growIfArrayFull() {
```